azure/templates/identity-service-templating.yml

```diff
@@ -3,16 +3,59 @@ steps:
     displayName: 'Use Python 3.8'
     inputs:
       versionSpec: 3.8
-
+        
   - template: ../components/setup-utils.yml
     parameters:
       serviceName: $(SERVICE_NAME)
 
+
+  - task: DownloadSecureFile@1
+    name: aws_config
+    inputs:
+      secureFile: ptl_aws_config
+
+  - bash: |
+      mkdir -p ~/.aws
+      mv $(aws_config.secureFilePath) ~/.aws/config
+      chmod 400 ~/.aws/config
+    displayName: "Get AWS config"
+
   - bash: |
       set -euo pipefail
+      
+      for secret_name in APIGEE_OTP_KEY \
+                         APIGEE_PASSWORD \
+                         ; do
+          secret_id="ptl/azure-devops/apigee-nonprod/${secret_name}"
+          secret_details=$(aws --profile apm_ptl secretsmanager get-secret-value --secret-id ${secret_id})
```

Comment:
no need to have two steps here .. use

aws --profile apm_ptl secretsmanager get-secret-value --secret-id ${secret_id} --query SecretString --output text